osf_tests/test_prereg.py

```diff
@@ -26,6 +26,7 @@ def test_not_logged_in(self):
                 'campaign_long': 'Prereg Challenge',
                 'campaign_short': 'prereg_challenge',
                 'is_logged_in': False,
+                'sign_up_url': 'http://localhost:5000/register/?campaign=prereg&next=http%3A%2F%2Flocalhost%2F',
```

Comment:
This is going to break if the `DOMAIN` setting is changed. Would it be better to use one of the derived base urls or just grab the parts of the string you need to verify?